docs/ops.html

```diff
@@ -3946,6 +3946,7 @@ <h3>Enter Migration Mode on the Brokers</h3>
   </p>
 
   <ul>
+    <li><a href="#brokerconfigs_broker.id">broker.id</a>: In zk mode, if <code>broker.id.generation.enable</code> is enabled (default is enabled), during migration, if <code>broker.id</code> is not set, the migration will fail because <code>node.id</code> will be -1. Both <code>broker.id</code> and <code>node.id</code> must be non-negative integers.</li>
```

Comment:
1. `In zk mode`: The section is all saying "zk mode" broker, no need to mention it again.
2. `if broker.id.generatio ...., if ....` : too many `if`. 
3. `the migration will fail because <code>node.id</code> will be -1` : we don't need to say this. Check the whole section again, we didn't say, `The controller.listener.name should also be added to listener.security.property.map because it will cause xxx error....`. Just tell users what to do in what condition.